Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -623,18 +623,21 @@ public void SendPlacementHints()
             LogDebug($"Hinting {hintedLocationIDs.Count()} locations.");
             try
             {
-                session.Socket.SendPacketAsync(new LocationScoutsPacket()
+                var locationScoutPacket = new LocationScoutsPacket()
```

Comment:
~~We should update this whole chunk of code to use the createAsHint option available in `ScoutLocationsAsync`.  (That option didn't exist at the time I originally wrote this)~~

The remarks of ScoutLocationsAsync suggest this might actually be somewhat problematic if we were to send out a second scout before the first returns, so keep it as-is for now.